contents/blog/sunsetting-helm-support-posthog.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2023-01-05
+title: "Sunsetting Kubernetes support for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["tim-glaser"]
+featuredImage: ../images/blog/green-blog-image.jpg
+featuredImageType: full
+categories: ["posthog-news"]
+---
+
+We're sunsetting support for our Kubernetes deployment for PostHog. Because we're an open source company, I want to be transparent about what this change means, and why we're making it. Around 3.5% of PostHog users currently use this type of deployment.
+
+## What changes
+
+We'll no longer be providing regular updates to our Helm chart. We'll also no longer be selling the Enterprise Self-Hosted plan. For at least the next 12 months we'll continue to provide security updates on the last available version.
+
+If you're currently using k8s to deploy PostHog, you have three options.
+
+The easiest option is to [migrate to PostHog Cloud]. We'll take care of everything, and it often works out much cheaper. For folks who had to self host to keep their data in the EU, we now offer PostHog EU Cloud. 
+
+You can also continue using PostHog as you are now. We'll still provide security updates. Unfortunately we'll no longer be able to provide community support for Kubernetes deployments. If you're a paid customer using Kubernetes, we'll contact you separately.
+
+If you have smaller volumes, you can migrate to our Docker deployment option. [PROVIDE LINK] It's much simpler to set up and manage.
+
+There will no longer be numbered releases of PostHog. Instead, we'll build a Docker image for each commit that happens in our GitHub repo. Just like on PostHog Cloud, folks using our Docker deployment can benefit immediately from new features and bugfixes. They'll also be able to pin a version or roll back where necessary.
+
+## Why are we doing this?
+
+We realized that our Kubernetes deployment did not work for both our free users and paying customers.
+
+Hosting PostHog at scale is complex. With our kubernetes users, we've seen issues crop up at every part of the stack. In event ingestion, Kafka, Clickhouse, Postgres, Redis and within the application itself. Sometimes the fix is simple ("increase disk space"), but often the issue is something a couple of layers deep and very hard to debug, involving long calls with expensive engineers on both sides.
```

Comment:
```suggestion
Hosting PostHog at scale is complex. With our Kubernetes users, we've seen issues crop up at every part of the stack. In event ingestion, Kafka, Clickhouse, Postgres, Redis and within the application itself. Sometimes the fix is simple ("increase disk space"), but often the issue is something a couple of layers deep and very hard to debug, involving long calls with expensive engineers on both sides.
```

Reply:
```suggestion
Hosting PostHog at scale is complex. With our Kubernetes users, we've seen issues crop up in every part of the stack. In event ingestion, Kafka, ClickHouse, Postgres, Redis and within the application itself. Sometimes the fix is simple ("increase disk space"), but often the issue is something a couple of layers deep and very hard to debug, involving long calls with expensive engineers on both sides.
```

Reply:
Is it worth mentioning partners here? That we tried working with them, impact on them, etc